### PR TITLE
Add testrunner Dockerfile

### PR DESCRIPTION
**Pull request by ravimeda**
Dockerfile that enables running PowerShell scripts.  This is needed for official builds.

### CODE REVIEW

**Comment by MichaelSimons** [comment]
Can you also make the `--pull` tweak to the build script that we discussed?

File: Dockerfile.linux.testrunner [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Use this Dockerfile to create a Linux testrunner image
```
Comment (MichaelSimons) [inline Dockerfile.linux.testrunner]: Thinking about the name, I think the term "test" is confusing.  I think the more abstract `runner` term is better.  This isn't being used for tests.

Comment (ravimeda) [reply Dockerfile.linux.testrunner]: Valid point.  I'll update the comment as - `# Use this Dockerfile to create a Linux runner image`

**Comment by ravimeda** [comment]
Yes, I will update `build.ps1` to include pull -
`& docker build -t imagebuilder --pull -f ./Dockerfile.linux.imagebuilder .`

**Pull request merged**